modin/engines/base/frame/data.py

```diff
@@ -1114,8 +1168,8 @@ def _fold_reduce(self, axis, func):
 
         Returns
         -------
-        BasePandasFrame
-            Pandas series containing the reduced data.
+            BasePandasFrame
+                Pandas series containing the reduced data.
```

Comment:
remove tab.
What is output type: `BasePandasFrame` or pandas Series?

Reply:
fixed